packages/contracts/contracts/test-fixtures/ResolveToPay5WeiApp.sol

```diff
@@ -11,26 +11,16 @@ import "../CounterfactualApp.sol";
 /// `Transfer.Transaction` object when the channel is closed.
 contract ResolveToPay5WeiApp is CounterfactualApp {
 
+  struct Resolution {
+    uint256 val;
+  }
+
   function resolve(bytes memory encodedState, Transfer.Terms memory terms)
     public
     pure
-    returns (Transfer.Transaction memory)
+    returns (bytes memory)
   {
-    uint256[] memory amounts = new uint256[](1);
-    amounts[0] = 5 wei;
-
-    address[] memory to = new address[](1);
-    to[0] = address(0);
-
-    bytes[] memory data;
-
-    return Transfer.Transaction(
-      0,
-      address(0),
-      to,
-      amounts,
-      data
-    );
+    return abi.encode(500);
```

Comment:
Why sometimes packed and sometimes not?

Reply:
Is this still relevant @IIIIllllIIIIllllIIIIllllIIIIllllIIIIll ?

Reply:
deleted